tls/s2n_recv.c

```diff
@@ -132,6 +132,7 @@ ssize_t s2n_recv(struct s2n_connection *conn, void *buf, ssize_t size, int *more
             if (r == -2) {
                 conn->closed = 1;
                 GUARD(s2n_connection_wipe(conn));
+		*more = 0;
```

Comment:
Change looks great. I think the indentation is just one char off (I hate being a whitespace or indentation nagger, but it's one thing we're consistent about). The '*' should start at the indentation boundary.